### DISCUSSION TITLE
nextjs example

### DISCUSSION THREAD

**Discussion by cborgia** [directus/directus#14891] [Feature Requests]
not sure if they (vercel) would invite this PR from you, or if they have to create it- but it would nice if directus had an official example for next-js  currently there are many other [example CMSs in their repo](https://github.com/vercel/next.js/tree/canary/examples) but not directus:

<img width="1347" alt="image" src="https://user-images.githubusercontent.com/55818/183156208-d6b72029-d27c-4f13-988c-51994364fb8f.png">

**Comment by rijkvanzanten** [comment]
Totally! Just a tad annoying to maintain, as it would have to be based on their template. We already have a nextjs example based on our own example structure here: https://github.com/directus/examples/tree/main/nextjs

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
